src/models/Review.py

```diff
@@ -0,0 +1,33 @@
+#temp
+from bson import ObjectId
+import datetime
+
+class Review:
+  _id: ObjectId
+  user_id: str
+  drink_id: str
+  comment: str # make the comment to be one singualr string
+  date = datetime.datetime.now()
+
+  def __init__(self, user_id: str, drink_id: str, comment: str) -> None:
+    """
+    Create a Review object according to our system diagram.
+    """
+    self.user_id = user_id
+    self.drink_id = drink_id
+    self.comment = comment
+
+  #need to come back to this funciton
+  def toJSON(self) -> dict[str, str]:
```

Comment:
Just discovered a bug in my own code while looking over the User model, so I figured I'd save us some time by sharing here.

For starters, `toJSON()` should just return a general dict because JSON can handle pretty much any simple data type. Putting a dict of strings as the return type, severely limits us and probably would've broken some things down the line.

`vars()` only grabs the str attributes of an object and returns them as key,value pairs in a dict. We'd be losing `_id` and `date`. I think you might have to handle those separately, put you can manually add them to `res` on line 23. JSON can't handle ObjectIds so can you convert that to a str before adding to res? Everything else should be good.

Reply:
Reviving this because I was wrong. `vars()` grabs all of the objects attributes, but it was only grabbing the str ones before because they were declared as _class-level_ attrs.

Also return type should just be `dict`. Now that I've caught the class-level attrs mistake, the majority of `toJSON()` should be handled internally by `vars()`. The only left is to convert special cases to JSON-compatible data-types (which you're already doing on L 24 - 26). That `for loop` can be removed.